packages/c/curaengine/package.yml

```diff
@@ -34,12 +34,12 @@ setup      : |
     # Remove bundled libraries
     rm -rf libs
 
-    %cmake_ninja -DSET_RPATH:BOOL=OFF \
```

Comment:
But why

Reply:
buggy cmake_ninja macro

```
+ ninja install -j16 -C solusBuildDir
ninja: Entering directory `solusBuildDir'
[0/1] Install the project...
/bin/sh: line 1: cd: /home/build/YPKG/root/curaengine/build/CuraEngine-5.4.0/solusBuildDir: Not a directory
FAILED: CMakeFiles/install.util 
cd /home/build/YPKG/root/curaengine/build/CuraEngine-5.4.0/solusBuildDir && /usr/bin/cmake -P cmake_install.cmake
ninja: build stopped: subcommand failed.
```

Reply:
same for libarcus. This happens sometimes and i don't know why